src/tools/perf/lib/ucp_tests.cc

```diff
@@ -294,7 +295,12 @@ class ucp_perf_test_runner {
 
         ucp_perf_test_prepare_iov_buffers();
 
-        m_perf.allocator->memset((char*)m_perf.recv_buffer + length - 1, -1, 1);
+        if (CMD == UCX_PERF_CMD_PUT) {
```

Comment:
CMD != UCX_PERF_CMD_AM

Reply:
this is for UCP, we don't want to use UCP RMA and AMO tests for 0-size messages
UCP AM/TAG/TAG_SYNC/STREAM can be used (they don't need to set `sn` to receive buffer)
change it

Changed to `if (length != 0)`